### PR TITLE
Fix address validation regexp in console

### PR DESCRIPTION
**Pull request by bafonins**
#### Summary

Closes https://github.com/TheThingsNetwork/lorawan-stack/issues/878

### CODE REVIEW

File: pkg/webui/console/lib/regexp.js [hunk 1]
```diff
@@ -15,7 +15,7 @@
 // alphanumeric, dashes, lowercase
 export const id = /^[a-z0-9]+(-[a-z0-9]+)*$/
 export const apiKey = /^NNSXS.[A-Z0-9]{39}.[A-Z0-9]{52}$/
-export const address = /^(?:(?:[a-zA-Z0-9]|[a-zA-Z0-9][a-zA-Z0-9\\-]*[a-zA-Z0-9])\\.)*(?:[A-Za-z0-9]|[A-Za-z0-9][A-Za-z0-9\\-]*[A-Za-z0-9])(?::[0-9]{1,5})?$|^$/
+export const address = new RegExp('^(?:(?:[a-zA-Z0-9]|[a-zA-Z0-9][a-zA-Z0-9\\-]*[a-zA-Z0-9])\\.)*(?:[A-Za-z0-9]|[A-Za-z0-9][A-Za-z0-9\\-]*[A-Za-z0-9])(?::[0-9]{1,5})?$|^$')
 export const latitude = /^[-+]?([1-8]?\d(\.\d+)?|90(\.0+)?)$/
-export const longitude = /^\s*[-+]?(180(\.0+)?|((1[0-7]\d)|([1-9]?\d))(\.\d+)?)$/
+export const longitude = /'^\s*[-+]?(180(\.0+)?|((1[0-7]\d)|([1-9]?\d))(\.\d+)?)$/
```
Comment (johanstokking) [inline pkg/webui/console/lib/regexp.js]: Is this ok?

Comment (bafonins) [reply pkg/webui/console/lib/regexp.js]: nope

**Comment by kschiffer** [comment]
> Can't we just get validation regexes directly from the API description that we generate for the console? https://github.com/TheThingsNetwork/lorawan-stack/blob/master/sdk/js/generated/api.json has the exact rules for each field. If we can derive console validation rules from that file on build, we don't have to change things in multiple places if this ever changes.
> 
> The format of API keys could for example change in [TheThingsIndustries/lorawan-stack#1666](https://github.com/TheThingsIndustries/lorawan-stack/issues/1666), which would break all API key fields in the console, while this wouldn't be caught in any tests.

I agree that this would be a lot better, But let's look into this in a separate issue. I'll create one. For the time being this fix will serve us well.

**Comment by bafonins** [comment]
> Can't we just get validation regexes directly from the API description that we generate for the console? 

I can imagine this not as trivial as i might seem.

**Pull request merged**